packages/landing/src/components/map.tsx

```diff
@@ -41,6 +40,8 @@ export class Basemaps extends Component<unknown, { isLayerSwitcherEnabled: boole
     const location = Config.map.location;
     this.map.setZoom(location.zoom);
     this.map.setCenter([location.lon, location.lat]);
+    if (location.bearing) this.map.setBearing(location.bearing);
```

Comment:
Does it matter if this doesn't get set? its much safer to do `!= null`

Reply:
Yeah, we should just set it when there is a value. It was never been set before this changes. I guess not set just not impact anything and remain the existing bearing of the map. The `setBearing` called when we change the bearing in url.

Reply:
Updated with `!= null`